package.json

```diff
@@ -128,14 +128,9 @@
     "node": ">=10.0.0"
   },
   "exports": {
-    ".": {
       "types": "./dist/rollup.d.ts",
-      "node": {
-        "require": "./dist/rollup.js",
-        "import": "./dist/es/rollup.js"
-      },
+      "require": "./dist/rollup.js",
+      "import": "./dist/es/rollup.js",
       "default": "./dist/es/rollup.browser.js"
-    },
-    "./dist/*": "./dist/*"
```

Comment:
it indeed is a breaking change to remove this - as import `rollup/dist/xxx` will throw an error. :)